### PR TITLE
:sparkles: Adds markdown language support

### PR DESCRIPTION
**Pull request by rickstaa**
I added markdown language support using the https://github.com/remarkjs/remark-lint
markdown linter.

### CODE REVIEW

File: fmts/markdown.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	register(&Fmt{
+		Name: "remark-lint",
+		Errorformat: []string{
+			`%P%f`,
+			`%#%l:%c %# %trror  %m`,
+			`%#%l:%c %# %tarning  %m`,
+			`%-Q`,
+			`%-G%.%#`,
+		},
+		Description: "Tool for writing clean and consistent markdown code.",
```
Comment (mgrachev) [inline fmts/markdown.go:15]:
```suggestion
		Description: "Tool for writing clean and consistent markdown code",
```

**Review by mgrachev — COMMENTED** [review]
@rickstaa 👋 Thank you for your contribution! 👍

**Review by rickstaa — COMMENTED** [review]
Good catch.

**Comment by lorenzo** [comment]
anything we can do to get this merged?

**Review by mgrachev — APPROVED** [review]

**Comment by mgrachev** [comment]
@rickstaa Thank you! ✌️

**Comment by review-dog** [comment]
Hi, @rickstaa! We merged your PR to reviewdog! 🐶
Thank you for your contribution! ✨

**We just invited you to join the @reviewdog organization on GitHub.**
Accept the invite by visiting https://github.com/orgs/reviewdog/invitation.
By joining the team, you'll be a part of reviewdog community and can help the maintainance of reviewdog.

Thanks again!

**Pull request merged**